uk.co.compucorp.civicrm.tasksassignments/js/controllers/taskList.js

```diff
@@ -227,21 +227,6 @@ define(['controllers/controllers',
                 });
             };
 
-            $scope.labelDateRange = function(from, until){
```

Comment:
Not anymore. It was used to display dates range in header. Now instead of dates we simply show 'Due Tasks:'.